client/WEB-INF/classes/resources/messages_es.properties

```diff
@@ -808,7 +808,7 @@ label.skip.guide=He utilizado CloudStack anteriormente, saltar esta gu\u00eda
 label.snapshot=Instant\u00c3\u00a1nea
 label.snapshot.limits=instant\u00c3\u00a1neas L\u00c3\u00admites
 label.snapshot.name=Nombre de instant\u00c3\u00a1neas
-label.snapshot.schedule=Lista de instant\u00c3\u00a1neas
+label.snapshot.s=instant\u00c3\u00a1neas
```

Comment:
Another commit from Mike (Setup != Set up)

https://git1-us-west.apache.org/repos/asf?p=cloudstack.git;a=blobdiff;f=client/WEB-INF/classes/resources/messages.properties;h=f81a196cf711d709af6a399a6f5e6d275dd7e951;hp=dcbf6c83a24a4cfd6977659d4f531c64c1786edb;hb=e640e0cf;hpb=c0230273cdbdf2558f4a0802d177bd5757de34fd

=> New value for a key on the source language file means remove the old translation (key/value) until a translator (re)translate the new value